### DISCUSSION TITLE
How disable some init ...

### DISCUSSION THREAD

**Discussion by straga** [Arksine/moonraker#635] [Q&A]
How too disable over-config? Or how to disable all, but enable only what is in config.
[dbus_manager]
[proc_stats]
[announcements]
[webcam]

**Comment by Arksine** [comment]
Those components are part of Moonraker's [core](https://moonraker.readthedocs.io/en/latest/configuration/#core-components) and will always be loaded.  If no configuration is provided they are loaded with default values.  Moonraker depends on these components and provides a guarantee to front ends that they will be available.

**Comment by straga** [comment]
Thanks a lot. The right way, fork and clean all hardcodes.
